core/services/subscription_test.go

```diff
@@ -123,3 +130,66 @@ func TestServices_ReceiveLogRequest_IgnoredLogWithRemovedFlag(t *testing.T) {
 		return count - originalCount
 	}).Should(gomega.Equal(0))
 }
+
+type mockEthSubscription struct{}
```

Comment:
I believe we already have one of these in the cltest package, EmptyEthMockSubscription it might be called.